### ISSUE TITLE
"Do you want to add a shortcut for the application xxx?" window is not listed in the taskbar area on Windows.

### ISSUE THREAD

**Issue by t-matsuu** [karakun/OpenWebStart#607]
It's a little difficult to get back to the JNLP apps if we use other apps while starting up the JNLP apps.

**Comment by Gnaneshwar-ux** [comment]
Yes this is real issue.. everytime I want to see console I have to search in recent windows, good if consoles shows up in taskbar. Thanks

**Comment by janakmulani** [comment]
Which version of OWS are you using?

**Comment by Gnaneshwar-ux** [comment]
I have OpenWebStart 1.13.0

It would be great if OWS show its icon on the taskbar when something like console runing in background. Currently it is difficult to track console windows.

<img width="1919" height="1079" alt="Image" src="https://github.com/user-attachments/assets/510f1080-e2b1-4637-bf4c-594024545d7e" />
